js/piwik.js

```diff
@@ -1403,7 +1403,17 @@ if (typeof Piwik !== 'object') {
             },
             hasNodeCssClass: function (node, klassName)
             {
-                if (node && klassName && node.className) {
+                if(!node || !klassName) {
+                    return false;
+                }
+
+                // For modern browsers cf http://caniuse.com/#search=classList
+                if(node.classList) {
+                    return node.classList.contains(klassName);
```

Comment:
Yup, if we check can I use we have only two issues for the partial support: 
- toggle second arg
- svg support

To test, why don't you use [browserstack](https://www.browserstack.com/list-of-browsers-and-platforms?product=automate) ?